charts/retool/templates/deployment_backend.yaml

```diff
@@ -153,7 +153,7 @@ spec:
 
           {{- include "retool.telemetry.includeEnvVars" . | nindent 10 }}
 
-          {{- if and (not .Values.externalSecrets.enabled) (not .Values.externalSecrets.externalSecretsOperator.enabled) }}
+          {{- if and (not .Values.externalSecrets.enabled) (not .Values.externalSecrets.externalSecretsOperator.enabled) (not .Values.externalSecrets.allowOtherSecrets) }}
```

Comment:
I think this logic is wrong. shouldn't it be
```
if or (not (or (externalSecrets.enabled) (esOperator.enabled))) allowOtherSecrets
```
which is roughly "if neither externalSecret source is enabled, or if they are but allowOtherSecrets is true"

Reply:
also would love to put this in a helper at this point 😄

Reply:
yeah it's wrong, i did this right before bed, will fix and put into a helper